controllers/front/ProductController.php

```diff
@@ -1117,7 +1117,7 @@ public function getTemplateVarProduct()
         $product['minimal_quantity'] = $this->getProductMinimalQuantity($product);
         $product['quantity_wanted'] = $this->getRequiredQuantity($product);
         $product['extraContent'] = $extraContentFinder->addParams(['product' => $this->product])->present();
-
+        $product['ecotax_tax_inc'] = $this->product->getEcotax(null, true, true);
```

Comment:
Yep I agree, but I think it is a BC, because `ecotax_tax_inc` attribute is not accessible anymore in presented product (consider the case it is used in miniature for example).
Can you please add BC label in this case please ?
Thanks

Reply:
I can move it to `Product::getProductProperties()`, that would eliminate the BC break I think @PululuK